numba/tests/test_array_reductions.py

```diff
@@ -307,6 +307,9 @@ def check(arr):
             got = cfunc(arr)
             self.assertPreciseEqual(got, expected)
 
+        # Empty array case
+        check(np.array[()])
```

Comment:
```suggestion
        check(np.array([]))
```

Seems like there's a typo.

Reply:
I see , I will commit that suggestion.